bittensor/_neuron/text/template_miner/neuron_impl.py

```diff
@@ -122,7 +122,7 @@ def run( self ):
             self.epoch = 0   
 
             # ---- reloads previous run if not restart ----
-            if self.config.neuron.no_restart:
+            if not self.config.neuron.no_restart:
```

Comment:
This is a very important change haha, we had been reloading previous runs before if no_restart = false but now we are reloading previous runs if no_restart = true?

Reply:
TD;LR, i think --no_restart is kind of messy, can we just use --restart?

because no_restart = False means that we should restart, no? 
I have also changed the logic on the default end that no_restart by default would be True (meaning it should not restart),
and users should configure it to restart with --no_restart False (meaning that it should restart) 
but eugene raised a problem where, bool type dosent work well with argparse, so that we should actually store_false for the --no_restart, 
but then when users type --no_restart, wouldnt they expect there is no restart?